Aquality.Selenium.Core/src/Aquality.Selenium.Core/Visualization/DumpManager.cs

```diff
@@ -30,6 +30,8 @@ public DumpManager(IDictionary<string, T> elementsForVisualization, string formN
         protected ILocalizedLogger LocalizedLogger { get; }
 
         protected string DumpsDirectory => VisualizationConfiguration.PathToDumps;
+        protected string ImageFormat => VisualizationConfiguration.ImageFormat;
```

Comment:
better to have it of type ImageFormat. It cannot be just ".abc", we have a list of possible values